pontoon/checks/tasks.py

```diff
@@ -0,0 +1,68 @@
+import logging
+
+import time
+
+from celery import shared_task
+
+from django.db import (
+    transaction
+)
+from pontoon.checks import libraries
+from pontoon.checks.utils import get_failed_checks_db_objects
+from pontoon.checks.models import (
+    Warning,
+    Error,
+)
+
+from pontoon.base.models import (
+    Translation,
+)
+
+log = logging.getLogger(__name__)
+
+
+@shared_task(bind=True)
+def bulk_run_checks(self, translations_pks):
+    """
+    Run checks on translations
+    :arg list[int] translations: list of primary keys for translations that should be processed
+    :arg int batch_size: Size of batch that should be checked at once by a worker
+    """
+    start_time = time.time()
+
+    with transaction.atomic():
+        translations = (
+            Translation.objects
+            .prefetch_related('entity', 'entity__resource__entities', 'locale')
+            .filter(pk__in=translations_pks)
+        )
+        warnings, errors = [], []
+        for translation in translations:
+            batch_warnings, batch_errors = get_failed_checks_db_objects(
+                translation,
+                libraries.run_checks(
+                    translation.entity,
+                    translation.locale.code,
+                    translation.entity.string,
+                    translation.string,
+                    False
```

Comment:
Maybe we should used named arguments, otherwise at least this `False` value is pretty cryptic?